packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -347,6 +348,10 @@ export class ApiReferencesTable extends React.Component<IApiReferencesTableProps
     return textElements;
   }
 
+  private _onShouldVirtualize(): boolean {
```

Comment:
Possibly make this a lambda just in case we end up adding more logic which uses `this` in the future? (same in other files)

Reply:
Sure, done in https://github.com/OfficeDev/office-ui-fabric-react/pull/9824/commits/1072054a2f9d15087723298cacdcaa05ebec8fdd.